blocks/init/src/Blocks/components/lists/lists-style.scss

```
@@ -1,26 +1,32 @@
 .lists {
 	margin: 0;
 	padding-left: 0;
-	color: var(--lists-color);
-	font-size: var(--lists-font-size);
-	line-height: var(--lists-line-height);
+	color: var(--lists-text-color, var(--lists-color, var(--global-colors-black, currentColor)));
+	font-size: var(--lists-font-size, 1rem);
+	line-height: var(--lists-line-height, 1.5);
 
 	li {
-		margin-left: 1.1em;
+		margin-left: 1.1rem;
+
+		&::marker {
+			color: var(--lists-marker-color, currentColor);
+		}
 	}
 
 	a {
 		cursor: pointer;
 		text-decoration: underline;
-		color: var(--lists-link-color);
+		color: var(--lists-link-color, var(--global-colors-primary, #5555CE));
+		text-decoration-color: var(--lists-link-color, var(--global-colors-primary, #5555CE));
```

Why not something like this instead of this literals?

```suggestion
		text-decoration-color: var(--lists-link-color, var(--global-colors-primary, var(--global-colors-yellow));
```

---

I wanted the default to be the "primary" color, if you don't have that one then you probably don't have the "yellow" either 😄  Just a hard fallback and can be removed if you think it's extra